nltk/sem/linearlogic.py

```diff
@@ -450,11 +446,7 @@ def __eq__(self, other):
         return self.d == other.d
 
     def __str__(self):
-        return (
-            "{"
-            + ", ".join("%s: %s" % (v, self.d[v]) for v in sorted(self.d.keys()))
-            + "}"
-        )
+        return "{" + ", ".join(f"{v}: {self.d[v]}" for v in sorted(self.d.keys())) + "}"
```

Comment:
something i learned recently, a lot of calls of dictionary `.keys()` is unnecessary. there's a funky pre-commit hook that undoes this https://github.com/mxr/unkey -- but i think we could also just call this hook on the whole nltk repo without actually adding it to pre-commit